Doc: Document the X-Grafana-Org-Id HTTP header (#32478)
grafana/grafana: docs/sources/http_api/auth.md

Currently you can authenticate via an `API Token` or via a `Session cookie` (acquired using regular login or OAuth).

> ## Header

Reviewer: ```suggestion
## X-Grafana-Org-Id Header
```

> **X-Grafana-Org-Id** is an optional property that specifies the org to which the action is applied on. If it is not set, the created key belongs to the current context org. You could use this header in all requests excepts requests regarding admin.

Reviewer 2: ```suggestion
**X-Grafana-Org-Id**  is an optional property that specifies the organization to which the action is applied. If it is not set, the created key belongs to the current context org. Use this header in all requests except those regarding admin.
```

  "secondsToLive": 86400
}
```
Header

> - **X-Grafana-Org-Id** - The org to which the created key belongs. It is optional. If it is not set, the created key would belongs to the current context org.

Reviewer 2: ```suggestion
- **X-Grafana-Org-Id** - An optional property that specifies the org to which the created key belongs. If it is not set, the created key belongs to the current context org.
```
